apps/wizard/app_pages/chart_diff/chart_diff.py

```diff
@@ -78,7 +87,7 @@ def slug(self) -> str:
         """
         if self.target_chart:
             assert self.source_chart.config["slug"] == self.target_chart.config["slug"], "Slug mismatch!"
-        return cast(str, self.source_chart.config["slug"])
+        return self.source_chart.config.get("slug", "no-slug")
```

Comment:
when is the case that a chart won't have a slug?

Reply:
I created a new draft chart for testing and it didn't have slug for some reason. It's an edge case which won't happen in real life, but I had to fix it for development.